resources/views/formfields/relationship.blade.php

```diff
@@ -148,7 +148,7 @@
 					class="form-control @if(isset($options->taggable) && $options->taggable == 'on') select2-taggable @else select2 @endif" 
 					name="{{ $relationshipField }}[]" multiple
 					@if(isset($options->taggable) && $options->taggable == 'on')
-						data-route="{{ route('voyager.'.str_slug($options->table).'.store') }}"
+						data-route="{{ route('voyager.' . $dataType->slug . '.store') }}"
```

Comment:
I think this is not right.

My setup:
* B1: slug='events-test'
* B1: slug='applicants'

When I dump the content of `$dataType->slug`, the output is: `applicants`. This should be `events-test`. 

```text
// Editing an applicant

@dump(@dump(route('voyager.'.str_slug($options->table).'.store'))) // => resulted in an expected error
@dump(route('voyager.'.$dataType->slug.'.store')) // => "http://sb-voyager-app.test/admin/applicants"
```

Reply:
Take a look here:
https://github.com/the-control-group/voyager/blob/1.1/src/Actions/ViewAction.php#L31

Reply:
I was able to solve it with: https://github.com/cyrildewit/voyager/blob/c558ede4f5d9a1f04f84277a91d527f2d1499472/resources/views/formfields/relationship.blade.php#L152

Reply:
Hum, I don't understand everything but when we tag an event in B2, we need to store the value in B1 (Slug: `event-test`), so that's the right value, isn't it?

Reply:
So if I put `Voyager::model('DataType')->where('name', $options->table)->first()->slug;` instead, it should work...

Reply:
I properly retested my solution and it works well... I don't understand what's the problem.

Reply:
Okay, both `Voyager::model('DataType')->where('name', $options->table)->first()->slug` and `$dataType->slug` works 🤔 
@cyrildewit 's solution use the same logic as `$option->table`. Mine use the opposite logic.